### ISSUE TITLE
Card description vanishes or checkbox untickes when ticking checkbox

### ISSUE THREAD

**Issue by Zocker1999NET** [nextcloud/deck#2601]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Describe the bug
Create a card with checkboxes, click on one to tick it. The whole description vanishes or it works but if you tick another one, the first one is not ticked anymore.

### Steps to reproduce
1. Create a new board with a new list
2. Create a card with the following description:
```
- [ ] test 1
- [ ] test 2
```
3. Tick one of both boxes and see how the description vanishes or continue to step 4
4. Tick the other and see how either the description vanishes or the first box is not ticked anymore.

### Expected behaviour
In step 3, tick the first box without having a chance of losing the whole description. In step 4, have both boxes ticked as expected.

### Actual behaviour
After step 3/4 the description might vanish. After step 4, the first box may not be ticked as before.

## Server configuration detail

**Operating system:** Linux 4.19.0-10-amd64 #1 SMP Debian 4.19.132-1 (2020-07-24) x86_64

**Webserver:** nginx/1.14.2 (fpm-fcgi)

**Database:** mysql 10.3.23

**PHP version:** 

7.3.19-1~deb10u1
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, sodium, standard, cgi-fcgi, mysqlnd, PDO, xml, apcu, bcmath, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, gmp, iconv, igbinary, imagick, imap, intl, json, exif, mysqli, pdo_mysql, apc, posix, readline, redis, shmop, SimpleXML, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, wddx, xmlreader, xmlwriter, xsl, zip, Phar, Zend OPcache

**Nextcloud version:** 19.0.4 - 19.0.4.2

**Updated from an older Nextcloud/ownCloud or fresh install:** upgraded since ~14

**Where did you install Nextcloud from:** zip from main homepage nextcloud.com

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.5.0
 - activity: 2.12.1
 - admin_audit: 1.9.0
 - apporder: 0.11.0
 - bruteforcesettings: 2.0.1
 - calendar: 2.0.4
 - checksum: 0.4.5
 - cloud_federation_api: 1.2.0
 - comments: 1.9.0
 - contacts: 3.4.2
 - contactsinteraction: 1.0.0
 - cospend: 1.0.5
 - dav: 1.15.0
 - deck: 1.2.1
 - external: 3.6.0
 - federatedfilesharing: 1.9.0
 - federation: 1.9.0
 - files: 1.14.0
 - files_automatedtagging: 1.9.0
 - files_external: 1.10.0
 - files_markdown: 2.3.1
 - files_pdfviewer: 1.8.0
 - files_rightclick: 0.16.0
 - files_sharing: 1.11.0
 - files_trashbin: 1.9.0
 - files_versions: 1.12.0
 - files_videoplayer: 1.8.0
 - firstrunwizard: 2.8.0
 - issuetemplate: 0.7.0
 - logreader: 2.4.0
 - lookup_server_connector: 1.7.0
 - mail: 1.4.1
 - metadata: 0.12.0
 - nextcloud_announcements: 1.8.0
 - notes: 3.6.4
 - notifications: 2.7.0
 - oauth2: 1.7.0
 - ocdownloader: 1.7.8
 - password_policy: 1.9.1
 - phonetrack: 0.6.4
 - photos: 1.1.0
 - polls: 1.5.7
 - privacy: 1.3.0
 - provisioning_api: 1.9.0
 - quota_warning: 1.8.0
 - ransomware_protection: 1.7.0
 - serverinfo: 1.9.0
 - settings: 1.1.0
 - sharebymail: 1.9.0
 - sociallogin: 3.4.2
 - socialsharing_email: 2.1.0
 - support: 1.2.1
 - suspicious_login: 3.2.1
 - systemtags: 1.9.0
 - tasks: 0.13.6
 - text: 3.0.1
 - theming: 1.10.0
 - twofactor_admin: 3.0.0
 - twofactor_backupcodes: 1.8.0
 - twofactor_gateway: 0.17.0
 - twofactor_nextcloud_notification: 2.3.0
 - twofactor_totp: 5.0.0
 - twofactor_u2f: 6.0.0
 - updatenotification: 1.9.0
 - viewer: 1.3.0
 - workflowengine: 2.1.0
Disabled:
 - encryption
 - files_readmemd
 - recommendations
 - spreed
 - survey_client
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "cloud.banananet.work"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "19.0.4.2",
    "overwrite.cli.url": "cloud.banananet.work",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "theme": "",
    "loglevel": 2,
    "maintenance": false,
    "app_install_overwrite": [],
    "log_type": "file",
    "twofactor_enforced": "true",
    "twofactor_enforced_groups": [
        "admin"
    ],
    "twofactor_enforced_excluded_groups": [],
    "memcache.local": "\\OC\\Memcache\\APCu",
    "memcache.distributed": "\\OC\\Memcache\\Redis",
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 0
    },
    "updater.release.channel": "stable",
    "lost_password_link": "disabled"
}
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

## Client configuration

**Browser:** Mozilla/5.0 (X11; Linux x86_64; rv:83.0) Gecko/20100101 Firefox/83.0

**Operating system:** Debian Bullseye/Sid

## Logs

<details><summary>Nextcloud log</summary>

```
no logs related to Deck issue
```
</details>

<details><summary>Browser log</summary>

JavaScript Console log of Firefox:
```
JQMIGRATE: Migrate is installed, version 1.4.1 jquery-migrate.min.js:2:551
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 2 globals.js:61:15
Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20 globals.js:61:15
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 3 globals.js:61:15
Notifications permissions granted App.vue:290
jQuery is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. globals.js:61:15
New search handler registered search.js:67:12
session heartbeat polling started session-heartbeat.js:97:9
$ is deprecated: The global jQuery is deprecated. It will be updated to v2.4 in Nextcloud 20 and v3.x in Nextcloud 21. In later versions of Nextcloud it might be removed completely. Please ship your own. 78 globals.js:61:15
Could not find capabilities initial state fall back to _oc_capabilities 17 index.js:14:12
OC.requestToken changed ***REMOVED SENSITIVE VALUE MANUALLY***
```
</details>

**Comment by stefan-niedermann** [comment]
Duplicate of https://github.com/nextcloud/deck/issues/2580, https://github.com/nextcloud/deck/issues/2591, #2595 and #2592 

[A fix](https://github.com/nextcloud/deck/pull/2584) is already in the pipeline.

You should be able to [restore the description content from the timeline tab](https://github.com/nextcloud/deck/issues/2580#issuecomment-730319105).

Please use the search function next time before open a new issue, thank you 🙂.

**Comment by Zocker1999NET** [comment]
sorry, I searched for similar issues but did not found them, maybe because they are closed, thank you for the hint how I can restore the data lost :+1:
